packages/react-components/react-menu/src/components/MenuItemRadio/MenuItemRadio.test.tsx

```diff
@@ -6,10 +6,11 @@ import { MenuItemRadio } from './MenuItemRadio';
 import { isConformant } from '../../common/isConformant';
 import { MenuListProvider } from '../../contexts/menuListContext';
 import type { MenuListContextValue } from '../../contexts/menuListContext';
+import { MenuItemRadioProps } from './MenuItemRadio.types';
 
 describe('MenuItemRadio', () => {
   isConformant({
-    Component: MenuItemRadio,
+    Component: MenuItemRadio as React.FunctionComponent<MenuItemRadioProps>,
```

Comment:
Why is this needed?

Reply:
It isn't actually, I was just copy pasting from other codes implementations. passing `MenuItemRadioProps` as a generic to `isConformant<MenuItemRadioProps>` should be enough